back/engines/commercial/insights/spec/acceptance/stats_inputs_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource "Stats - Inputs" do
+  explanation "The various stats endpoints can be used to show certain properties of inputs."
+
+  before { header 'Content-Type', 'application/json' }
+
+  let(:view) { create(:view) }
+  let(:view_id) { view.id }
+  let(:ideas) { create_list(:idea, 5, project: view.scope) }
+  let(:other_ideas) { create_list(:idea, 4) }
```

Comment:
(Tiny detail: Agreed it's difficult to find the right spot here, but I'd reduce those count to reduce the execution time here. Maybe only 1 other idea is sufficient and 3-4 ideas for the view. And we can also move them to the 'when admin' to avoid creating them for 'unauthorized requests'.)